internal/distro/distro.go

```diff
@@ -40,25 +40,37 @@ type Distro interface {
 	Runner() string
 }
 
-var registered map[string]Distro
+type DistroRegister struct {
+	distros map[string]Distro
+}
 
-func init() {
-	registered = map[string]Distro{
-		fedora30.Name: fedora30.New(),
-		rhel82.Name:   rhel82.New(),
+func NewDistroRegister(confPaths []string) *DistroRegister {
```

Comment:
I really prefer `NewRegistry` (`Register` → `Registry` and no stuttering), but really don't worry changing it everywhere if that's too much hassle. It's just a preference, after all.

Reply:
Ah, you are totally right. Will fix.